example/public/index.html

```diff
@@ -115,6 +115,18 @@ <h1>Display BPMN 2.0 Diagrams Using React</h1>
         />
         <hr />
 
+        <ReactBpmn
+          url="/public/diagram.bpmn"
```

Comment:
Q: Why not simply provide the diagram from external source, i.e.

```
<ReactBpmn diagram={ someDiagramXMLPromise } ... />
```

Alternative is to implement your own component for that use-case. What keeps you from doing that?

Reply:
Didn't know the diagram param can take a promise. I'll try this and the suggested options. Thanks

Reply:
I did not imply this was an option, I simply think in different ways to solve your problem. :)

Reply:
Well just had to write a simple component. I was having issues with importing bpmn-js  in NextJs. importing from the production build like in this component library solved the problem.

This import works
```js
import BpmnJS from 'bpmn-js/dist/bpmn-navigated-viewer.production.min.js'
```

This import failed with several errors:
```js
import BpmnViewer from 'bpmn-js/lib/NavigatedViewer'
```